CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Next release
```

Comment:
We normally write this as part of the release process, to make sure we don't miss things. Let's drop it from this PR>

Reply:
Done.